draft-ietf-teas-actn-poi-applicability.md

```diff
@@ -482,9 +487,7 @@ respective P-PNC;
 - technology-specific mechanisms (in the case of inter-domain SR-TE,
 the binding SID) are used for the inter-domain TE path stitching;
 
-- each packet domain in {{fig-vpn-topo}} uses technology-specific local
-protection mechanisms (in the case of SR-TE, TI-LFA), with the
-awareness of multi-layer TE path properties (e.g., SRLG).
+- each packet domain in {{fig-vpn-topo}} uses technology-specific local protection mechanisms (such as Fast Reroute (FRR) in case of MPLS-TE or Topology Independent Loop-free Alternate Fast Reroute (TI-LFA) in case of SR-TE), with the awareness of multi-layer TE path properties (e.g., SRLG).
```

Comment:
```suggestion
- each packet domain in {{fig-vpn-topo}} uses technology-specific local protection mechanisms (such as Fast Reroute (FRR) in case of MPLS-TE or Topology Independent Loop-free Alternate Fast Reroute (TI-LFA) in case of SR-TE), with the awareness of multi-technologyTE path properties (e.g., SRLG).
```